inform7/Internal/Inter/Architecture32Kit/Sections/Input Output.i6t

```diff
@@ -319,6 +321,10 @@ to document all of that.
         glk_put_char_stream(gg_commandstr, 10); ! newline
     }
   .KPContinue;
+    for ( ix = 1 : ix <= (a_buffer-->0) : ix++ ) {
+      for ( jy = 1 : jy <= (UnicodeWhitespace-->0) : jy++ )
```

Comment:
Before checking against the list, I'd add some checks for the ranges we know won't match. That will be much more efficient than checking against all the list entries.

Reply:
Good point. Heck, I can just use `< 32` and eliminate the whole range of non-printable ASCII characters. I'll update the PR tomorrow.